cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/transform/ValidatorTransform.java

```diff
@@ -66,8 +65,6 @@
     "      var isValid = true; " +
```

Comment:
The String SCRIPT_DESCRIPTION needs to be updated to reflect the changes in the RST file.